layouts/partials/faq.html

```diff
@@ -0,0 +1,245 @@
+<!--==========================  faq area end  ==========================-->
+<section class="faq-area section-padding">
+  <div class="container">
+    <div class="columns is-vcentered ">
+      <div class="column is-6">
+        <div class="faq-left">
+
+           <div class="faq-image mb-40">
+             <img src="/assets/images/illustrations/faq.png" alt="faq-image">
+           </div>
+
+          <h2 class="title">frequently asked questions</h2>
+          <p>Find your answers here. if you don’t find it here, please contact us.</p>
```

Comment:
~Find your answers here. if you don’t find it here, please contact us.~
Replace it with:

Here are a few of the questions we get the most. If you don't see what's on your mind, contact us anytime.